src/components/BannerSlider/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import { LeftArrow, RightArrow, useInterval } from "@src/shared";
+import classNames from "classnames";
+
+interface BannerMockType {
+  img: string;
+  link: string;
+  text: string;
+}
+
+const bannerMock: BannerMockType[] = [{
+  img: "pink",
+  link: "catalog/pink",
+  text: "Go here and buy it PINK!",
+},
+{
+  img: "green",
+  link: "catalog/green",
+  text: "Go here and buy it GREEN!",
+},
+{
+  img: "blue",
+  link: "catalog/blue",
+  text: "Go here and buy it BLUE!",
+}];
+
+export default function BannerSlider (): JSX.Element {
+  const [bannerIndex, setBannerIndex] = useState(0);
+  const [banner, setBanner] = useState(bannerMock[0]);
+
+  function direction (direction: number): number {
```

Comment:
direction is not good name for function. What it does? It returns index of our new banner. So, user press like "back" or "forward", and based on this information - return smth.
I think it will be better to name it like getDirection/setDirection and put as parameter string "back" or "forward" instead of number - it is more verbose and understand.
    const s(g)etDirection = (direction: "back" | "forward"): number => {...}